runtime/oti/j9nonbuilder.h

```diff
@@ -5004,6 +5004,10 @@ typedef struct J9VMThread {
 #endif /* OMR_GC_COMPRESSED_POINTERS */
 #endif /* OMR_GC_CONCURRENT_SCAVENGER */
 	UDATA safePointCount;
+#if JAVA_SPEC_VERSION >= 16
+	UDATA *javaArgs;
```

Comment:
These aren't great names. I thought this was a thread local copy of the arguments to the java executeable.

Reply:
Renamed to `ffiArgs` and `ffiArgCount` which are specific to the parameters passed over to `ffi_call`.